snapper.spec.in

```diff
@@ -188,7 +194,13 @@ Authors:
 /sbin/ldconfig
 
 %package -n libsnapper-devel
+%if 0%{?suse_version} > 1325
+Requires:       libboost_system-devel
+Requires:       libboost_thread-devel
+Requires:       libboost_test-devel
```

Comment:
The boost test framework should not be needed to use/link the library since it is just used to run the unit tests,